### PR TITLE
Added configuration for FIPS enable in OCP deployment

### PR DESCRIPTION
**Pull request by gauravpbankar**
Signed-off-by: gauravpbankar <gauravpbankar@gmail.com>

### CODE REVIEW

File: var.tfvars [hunk 1]
```diff
@@ -47,6 +47,7 @@ cluster_id_prefix = "test-ocp" # Set it to empty if just want to use cluster_id
 cluster_id        = ""         # It will use random generated id with cluster_id_prefix if this is not set
 
 use_zone_info_for_names = true # If set it to false, the zone info would not be used in resource names on PowerVS.
+fips_compliant      = false
```
Comment (Prajyot-Parab) [inline var.tfvars:50]: This should be commented out, as by default it is already false. Also add a comment as seen for other options.

File: modules/5_install/variables.tf [hunk 2]
```diff
@@ -34,7 +34,7 @@ variable "region" {}
 variable "zone" {}
 variable "name_prefix" {}
 variable "node_prefix" {}
-
+variable "fips_compliant" {}
```
Comment (Prajyot-Parab) [inline modules/5_install/variables.tf:37]: don't remove the newline instead add it on a newline (maintain existing formatting for better distinction of diff options)

File: var.tfvars (hunk 1 above)
Comment (gauravpbankar) [reply var.tfvars]: Done

**Comment by gauravpbankar** [comment]
> "install_playbook_tag" in variables.tf need to be updated to the version which support FIPS.

Done

**Review by cs-zhang — APPROVED** [review]

File: docs/var.tfvars-doc.md [hunk 3]
```diff
@@ -203,6 +203,14 @@ The total length of `cluster_id_prefix`.`cluster_id` should not exceed 32 charac
 
 The `use_zone_info_for_names` is a flag to indicate whether to use `cluster_id`-`ibmcloud_zone` or only `cluster_id` as name prefix for resource naming on PowerVS. The default value is set to `true` to use zone info in names, and the total length of `cluster_id_prefix`-`cluster_id`-`ibmcloud_zone` should not exceed 32 characters.
 
+### FIPS Variable for openshift deployment
```
[the remark this answers is not in the document]
Comment (gauravpbankar) [reply docs/var.tfvars-doc.md]: Done

File: docs/var.tfvars-doc.md [hunk 4]
```diff
@@ -203,6 +203,14 @@ The total length of `cluster_id_prefix`.`cluster_id` should not exceed 32 charac
...
+
+These variables will be used for deploying ocp in FIPS mode.
```
[the remark this answers is not in the document]
Comment (gauravpbankar) [reply docs/var.tfvars-doc.md]: Done

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Pull request merged**